File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
      Map <Integer,Integer> m = new HashMap<>();
     int a;
     for(int i = 0; i<nums.length; i++){
      a = target-nums[i];
      if(m.containsKey(a)){
        return new int [] {m.get(a), i};
      }
      m.put(nums[i], i);
     }
     return new int [] {};
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        co = 0
        xc = 0
        for i in nums:
            if i == 1:
                co += 1
                xc = max(xc, co)
            else:
                co = 0    
        return xc    
        